hidimstat/test/test_ensemble_clustered_inference.py

```diff
@@ -45,10 +40,10 @@ def test_ensemble_clustered_inference():
                                      n_rand=n_rand)
 
     expected = 0.5 * np.ones(n_features)
-    expected[:n_support] = 0.0
-
-    for i in np.arange(n_support):
-        assert_almost_equal(sf_corr[i], expected[i], decimal=2)
+    expected[:support_size] = 0.0
 
-    for i in np.arange(n_support + margin_size, expected.size):
-        assert_almost_equal(sf_corr[i], expected[i], decimal=2)
+    assert_almost_equal(sf_corr[:support_size-margin_size],
+                        expected[:support_size-margin_size])
+    assert_almost_equal(sf_corr[support_size+margin_size:200],
```

Comment:
Beware pep8. You should rely on a linter to make syntax enforcement automatic.

Reply:
(I do)

Reply:
I will add spaces.